packages/analytics-v1.1/src/core/analytics.js

```diff
@@ -683,7 +683,7 @@ class Analytics {
       (options = properties), (properties = name), (name = null);
     if (typeof category === 'string' && typeof name !== 'string')
       (name = category), (category = null);
-    if (this.sendAdblockPage && category != 'RudderJS-Initiated') {
+    if (this.sendAdblockPage && category != 'RudderJS-Initiated' && '__RS_GOOGLE_ADS_SDK_URL__') {
```

Comment:
I'd skip the URL check here and add it in `sendSampleRequest` as this function needn't bother about it.

Reply:
sure, can do that in case we use sendSampleRequest from another part of the code too

Reply:
done